data/json/portal_storm_effect_on_condition.json

```diff
@@ -254,7 +234,7 @@
       { "remove_active_mission": "MISSION_INVESTIGATE_PORTAL_DUNGEON" },
       { "remove_active_mission": "MISSION_INVESTIGATE_PORTAL_STORM_CENTER" },
       { "alter_timed_events": "portal_dungeon_entrance" },
-      { "u_adjust_var": "portal_storm_counter", "type": "counter", "adjustment": 1 },
+      { "math": [ "portal_storm_counter", "++" ] },
```

Comment:
AFAIK, this is a `u_val`, and to provide backward compatibility, a proper naming like `u_counter_portal_storm_counter` is needed. ( The `npctalk_var` part is omitted when used in `math`.) The line 34 at data/json/npcs/isolated_road/isolated_road_jay_convert.json is an example, which was rewritten by me from `"u_add_var": "gunsmith_ammo_ammount", "type": "number", "context": "artisans", "value": "800" ` to `"math": [ "u_number_artisans_gunsmith_ammo_amount", "=", "800" ] `.

I think other u_val should also be taken care of.

Reply:
Am i understand correctly, that `u_cell_time` and `"u_val": "cell_time"` is the same thing from the code perspective? because apparently cell_time still has a lot of EoC that uses it as `{ "u_val": "cell_time", "default": "1 minutes" }`, and i don't know could it cause some problem, or its okay to use both `math` and `arithmetic` simultaneously

Reply:
Yes, I believe u_cell_time and "u_val": "cell_time" is the same thing.